packages/esm-patient-search-app/src/ui-components/overflow-menu.component.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect, useRef } from 'react';
```

Comment:
Hm this was just copied from patient chart, right? Maybe we should make this reusable, e.g., by making it an extension or placing it in the components lib.